### PR TITLE
Adding the NAT meetup and squishing python-js-ruby

### PR DESCRIPTION
**Pull request by MandyMeindersma**
## Do these code changes work locally and have you tested that they fix the issue yourself?

-   [ ] yes!

## Does the following command run without warnings or errors?

-   [ ] yes! `npm run pr-checks`

## Have you taken a look at our [contributing guidelines](https://github.com/devedmonton/DES-Website/blob/main/.github/CONTRIBUTING.md)?

-   [ ] yes!

## My node version matches the one suggested when running `nvm use`?

-   [ ] yes!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Note that all of these commands have been tested and used on Mac/unix machines,
 $ nvm use
```
Comment (deejayjay) [inline README.md:19]: @MandyMeindersma: Thanks for fixing the `npm run install` command. The `nvm use` command alone does nothing on my machine even though I have nvm installed on it. I think the set up instructions need to be updated a bit still. Can be done in a separate PR.

File: constants/meetups.ts [hunk 2]
```diff
@@ -1,29 +1,11 @@
 export const meetups = [
   {
-    name: 'YEGrb',
-    description: `YEGRB is Edmonton’s local Ruby Meetup. Ruby is a dynamic, open source programming language with a focus on simplicity and productivity. It has an elegant syntax that is natural to read and easy to write.`,
+    name: 'Javascript, Ruby and Python Meetup',
+    description: `A combination of the YEGRB, Exchange.js, and Edmonton.py community! A polyglot meetup that focusses on cool JS, Ruby and Python things. Both student's and working professionals welcome!`,
```
Comment (deejayjay) [inline constants/meetups.ts:4]: Found a couple of typos and grammatical error. How about rephrasing to this:
`A combination of the YEGRB, Exchange.js, and Edmonton.py communities! A polyglot meetup that focuses on cool JavaScript, Ruby, and Python topics. Both students and working professionals are welcome!`

**Review by deejayjay — CHANGES_REQUESTED** [review]
@MandyMeindersma: I have requested fixing a couple of typos and a grammar error. Please fix whenever you can and I will approve.

File: README.md (hunk 1 above)
Comment (MandyMeindersma) [reply README.md:19]: image


Oh weird. Mine worked yesterday?

File: constants/meetups.ts (hunk 2 above)
Comment (MandyMeindersma) [reply constants/meetups.ts]: hahahah thank you, I had a spelling mistake at work, on a public endpoint that got merged into production.... so it's something I am working on :P

File: README.md (hunk 1 above)
Comment (deejayjay) [reply README.md:19]: I'm using Windows. I believe `nvm use` expects a node version when we use it in Windows.

**Comment by deejayjay** [comment]
@MandyMeindersma I'm not sure why I didn't receive notifications when you pushed new changes. I'll approve the PR tomorrow morning 🙂.

**Review by deejayjay — APPROVED** [review]

**Comment by deejayjay** [comment]
@MandyMeindersma: Approved the PR.

**Pull request merged**